src/services/usePlaceService.ts

```diff
@@ -32,7 +32,7 @@ const usePlaceService = () => {
 			getPlaceSuggestions: async (QueryText: string) => {
 				const input: SuggestCommandInput = {
 					QueryText,
-					BiasPosition: [viewpoint?.longitude as number, viewpoint?.latitude as number],
+					BiasPosition,
```

Comment:
Dumb question - how does this work? Isn't the variable named "biasPosition", not "BiasPosition"?

Reply:
At line number 25, I have this
`const { mapPoliticalView, biasPosition: BiasPosition } = useMap();`
which is basically de-structuring biasPosition from `useMap` and renaming it to `BiasPosition`

Reply:
At line number 25, I have this
`const { mapPoliticalView, biasPosition: BiasPosition } = useMap();`
which is basically de-structuring biasPosition from `useMap` and renaming it to `BiasPosition`